lib/TTT/comp_player.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Comp_player do
```

Comment:
I would take a look at this regarding module names: https://github.com/elixir-lang/elixir/blob/master/lib/elixir/pages/Naming%20Conventions.md

Reply:
👍 thanks!